### ISSUE TITLE
[Issue]: Can't open books if "Allow media downloads" is disabled

### ISSUE THREAD

**Issue by linux2good** [jellyfin/jellyfin#8187]
### Please describe your bug

When you opening book on the account with disabled media downloading (option Allow media downloads), it goes to kind of "boot loop". I think the problem is because when you opening book, Jellyfin tries to **download** book with call to API, then, API checks if account is allowed to download media (no, its not allowed, Allow media downloads is disabled), then returns Bad Request (if not allowed) to client. Client should (somehow) stream books like videos to prevent this, but its probably hard to implement (with videos a lot easier, probably just parse bytes until EOF), or, make a new API call specially for books (some bookopen() ). I can't make a PR because i don't know C# (which is backend) or JavaScript. (BTW: I tested it with PDF and EPUB, none of this works)

### Jellyfin Version

Other

### if other:

10.8.1

### Environment

```markdown
- OS: Debian
- Virtualization:
- Clients: Firefox, Jellyfin Android
- Browser: Firefox
- FFmpeg Version: 5.0.1-Jellyfin
- Playback Method: Clicking on the book
- Hardware Acceleration: None
- Plugins: Defaults, and BookShelf
- Reverse Proxy: Nginx
- Base URL:
- Networking:
- Storage: local
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ppittle** [comment]
This is an interesting issue.  I was about to suggest simply marking this issue as an enhancement as the UI instructions for **Allow media downloads** states:
> Book libraries require [Allow media downloads] enabled to function properly.

![image](https://user-images.githubusercontent.com/1190907/184568184-93aeaa1c-df65-4b7d-a3a4-69fe55ca8df9.png)

However, digging through the issue history, 10.7 apparently added support for reading pdfs in browser:
https://features.jellyfin.org/posts/79/bug-suggestion-e-books-issues

![image](https://user-images.githubusercontent.com/1190907/184568788-05963551-52d3-4cea-be25-48772b941187.png)

So maybe this is a bug and the UI language is out of date?

**Comment by ppittle** [comment]
@thornbill  - could you weigh in on this issue?  Is this a bug or enhancement request?

**Comment by thornbill** [comment]
No. We don't do that here. https://jellyfin.org/docs/general/faq#when-will-release-x-happen-when-will-feature-x-be-available
